Controllers/Controller.php

```diff
@@ -56,9 +56,9 @@ public function __construct()
      * @param string $page Name of the view loaded
      * @param array  $args Options for Twig's rendering
      *
-     * @return Result generated by Twig
+     * @return mixed Result generated by Twig
```

Comment:
Pas de `@return mixed`, sois plus précis `@return string the result generated by Twig`

Reply:
Ok j'avaispas vu :/